cypher/cypher-docs/src/docs/dev/ql/administration/security/dbms/role-management-syntax.asciidoc

```diff
@@ -15,6 +15,12 @@ GRANT DROP ROLE
     TO role[, ...]
 | Enable the specified roles to delete roles.
 
+| [source, cypher]
```

Comment:
Once we redo this PR with the new syntax, don't forget to update all new [source, cypher] that should not be playable to [source, cypher, role=noplay]